ledger/error/src/main/scala/com/daml/error/definitions/LoggingPackageServiceError.scala

```diff
@@ -10,15 +10,18 @@ import com.daml.lf.data.Ref.PackageId
 import com.daml.lf.engine.Error
 import com.daml.lf.{VersionRange, language, validation}
 
-abstract class PackageServiceError(
+trait PackageServiceError extends BaseError
+
+abstract class LoggingPackageServiceError(
     override val cause: String,
     override val throwableO: Option[Throwable] = None,
 )(implicit override val code: ErrorCode)
-    extends BaseError.Impl(cause, throwableO) {
+    extends BaseError.Impl(cause, throwableO)
+    with PackageServiceError {
   final override def logOnCreation: Boolean = true
 }
 
-object PackageServiceError extends LedgerApiErrors.PackageServiceErrorGroup {
+object LoggingPackageServiceError extends LedgerApiErrors.PackageServiceErrorGroup {
```

Comment:
This will affect the generated docs as in 
```
1.1.3. ParticipantErrorGroup / LedgerApiErrors / PackageServiceError
```,
correct?

Reply:
Oh, no. This one shouldn't change. reverted